scripts/HPC_chatbot_preprocessor/parsed_mds/generic/compiling_your_software/compiling_your_software_paragraph_1.txt

```diff
@@ -0,0 +1,39 @@
+Compiling and testing your software on the HPC
+All nodes in the HPC cluster are running the "RHEL 8.8 (accelgor, doduo, donphan, gallade, joltik, skitty)" 
```

Comment:
@wdpypere  we need to take care of crap like this during the migration (or after migration is finished).

Reply:
we used to have a macros where we could fix all that in one place. But I will be updating docs yes.